main_test.go

```diff
@@ -15,19 +15,15 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-var (
-	ag   *autographer
-	mo   *monitor
-	conf configuration
-)
+func MockAutographer(t *testing.T) (*autographer, configuration) {
```

Comment:
Spent a significant time arguing with myself over:
1. The name of this function
2. Bringing in a test framework that would give us more of a concept of `fixtures` or `beforeEach`

Open to changing course on either honestly.

Reply:
Hm, yeah, seems to not really be a fake nor a mock. Unfortunate that they made this restricted constructor instead of making the real constructor safe for testing.

`testableAutographer`? `newTestAurographer`? Maybe the latter

Reply:
Yeah that sounds better.